R/add.segs.R

```diff
@@ -187,18 +226,15 @@ calculate.coords.dendrogram <- function(
         }
 
     dy <- x[, length.colname];
-    x.length <- v[x$tip, 'x.length']
+    x.length <- v[x$tip, 'x.length'];
     dx <- if (is.na(x.length)) x[, 'length'] * tan(angle) else x.length;
 
     offset.x <- offset * offset.x.modifier;
-
     basex <- basex + dx + offset.x;
+    # basex <- basex + offset.x;
```

Comment:
Any reason to keep this? I don't think we should leave commented-out code here as it will probably be unclear in the future.

Reply:
Oops yea, I probably just missed it while cleaning up the code